actionview/lib/action_view.rb

```diff
@@ -58,6 +58,7 @@ module ActionView
     autoload_at "action_view/template/resolver" do
       autoload :Resolver
       autoload :PathResolver
+      autoload :FileSystemResolver
```

Comment:
autoloading this is now required. It wasn't necessary before because AV::Base was autoloading it at boot time.
Now, ActionView Resolver caching make use of it before AV::Base get autoloaded